internal/config/testdata/good-0.yaml

```diff
@@ -22,8 +22,8 @@ database:
       hostname: "read-localhost"
 metrics:
   client_type: "noop"
-prometheus_metrics:
-  enabled: false
+  prometheus:
+    enabled: false
```

Comment:
We'll want to rename this to `enable`. This works because the default value is false anyway.

Reply:
thanks for catching